src/c++/library/common.h

```diff
@@ -227,6 +227,17 @@ struct InferOptions {
 ///
 class InferInput {
  public:
+  struct Options {
+    virtual ~Options() = default;
+  };
+
+  struct HttpOptions : public Options {
+    bool json_data = false;
```

Comment:
Perhaps name the option `binary_data=true` for parity with Python client naming?

Reply:
Changed 👍